### DISCUSSION TITLE
right sidebar (table of content) disappeared

### DISCUSSION THREAD

**Discussion by StefanoCecere** [vuepress-theme-hope/vuepress-theme-hope#4171] [Q&A] [answered]
since today, i don't see it anymore. i have to zoom out the page, to see it.
at "actual size" 100% it isn't visible...

**Comment by Mister-Hope** [comment]
Can you provide a valid reproduction?

Please note that the latest theme still supports rc9  of VuePress,  And we will plan to start rc 12 once it's released as next version.

So you should make sure that there isn't any peerdependencies warnings in your installation

**Reply by StefanoCecere** [reply]
the problem is in your theme site!
for example here are the same screen at 100% and at 90%.. as you can see at 100% the right sidebar doesn't appear (it's a macbook pro.. both in Firefox and CHrome)

![Screenshot 2024-05-21 at 04 52 52](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/640536/7f416b44-9007-4dcd-83e6-fd73792a1c98)


![Screenshot 2024-05-21 at 04 53 05](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/640536/91508b3e-a7e7-4a46-8724-4f44b97c573b)

**Reply by Mister-Hope** [reply, marked as answer]
This is because we use different layout with different device based on screen width.  You can still customize these breakpoints, see https://theme-hope.vuejs.press/faq/common-question.html#foldable-sidebar-toc-on-mobile-left-sidebar-on-pc

**Reply by Mister-Hope** [reply]
The outline content of this page will be a foldable one on the top of the page if you are using a tablet.  It is on the right only when the screen has enough space

**Comment by StefanoCecere** [comment]
ok, thank you. it seems strange that until last week it worked (showed the sidebar) now no. but maybe it's an update to macOS that now as default res for a retina 14inch M2 has 1352x878 resolution.. going higher res, it works.
thank you
